NoSuchMethodError: old Apache library loaded in Gradle build environment

<!--
Before filing an issue, please reproduce with the latest version of Jib.

If you are encountering errors pulling or pushing to remote registries,
please check our Frequently Asked Questions before filing an issue:

    https://github.com/GoogleContainerTools/jib/blob/master/docs/faq.md#registry-errors
-->

**Environment**:

- *Jib version:* 1.8.0
- *Build tool:* Gradle 5.4.1
- *OS:* OSX 10.14.6

**Description of the issue**:
When trying to run `gradle jibDockerBuild` with the jib gradle plugin v1.8.0, I got a very confusing error message with no results on Google:
```
> com.google.cloud.tools.jib.plugins.common.BuildStepsExecutionException: org.apache.http.client.config.RequestConfig$Builder.setNormalizeUri(Z)Lorg/apache/http/client/config/RequestConfig$Builder;
```

**Expected behavior**:
~~When running the same command with the jib plugin v1.7.0, I get this much more helpful error message:~~

**Steps to reproduce**:
<!-- Please provide a minimal and precise series of steps -->

  ~~1. Kill the docker daemon~~
  2. Add the jib gradle plugin v1.8.0 to a gradle build
  3. Run `gradle jibDockerBuild`

**`jib-gradle-plugin` Configuration**: <!-- Delete this section if not used -->
```groovy
plugins {
    id 'com.google.cloud.tools.jib' version '1.8.0'
}

```

<!-- Thanks for contributing! -->

here is the relevant portion of the stacktrace:
```
Caused by: java.lang.NoSuchMethodError: org.apache.http.client.config.RequestConfig$Builder.setNormalizeUri(Z)Lorg/apache/http/client/config/RequestConfig$Builder;
        at com.google.api.client.http.apache.v2.ApacheHttpRequest.<init>(ApacheHttpRequest.java:43)
        at com.google.api.client.http.apache.v2.ApacheHttpTransport.buildRequest(ApacheHttpTransport.java:179)
        at com.google.api.client.http.apache.v2.ApacheHttpTransport.buildRequest(ApacheHttpTransport.java:58)
        at com.google.api.client.http.HttpRequest.execute(HttpRequest.java:872)
        at com.google.cloud.tools.jib.http.FailoverHttpClient.call(FailoverHttpClient.java:309)
        at com.google.cloud.tools.jib.http.FailoverHttpClient.call(FailoverHttpClient.java:237)
        at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:139)
        at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:115)
        at com.google.cloud.tools.jib.registry.RegistryClient.callRegistryEndpoint(RegistryClient.java:473)
        at com.google.cloud.tools.jib.registry.RegistryClient.pullManifest(RegistryClient.java:303)
        at com.google.cloud.tools.jib.registry.RegistryClient.pullManifest(RegistryClient.java:307)
        at com.google.cloud.tools.jib.builder.steps.PullBaseImageStep.pullBaseImage(PullBaseImageStep.java:205)
        at com.google.cloud.tools.jib.builder.steps.PullBaseImageStep.call(PullBaseImageStep.java:126)
        at com.google.cloud.tools.jib.builder.steps.PullBaseImageStep.call(PullBaseImageStep.java:59)
```
and, to clarify, this is a subproject, but the parent project has no relevant dependencies. I'm also not upgrading, but adding jib for the first time. Downgrading to `1.7.0` and starting Docker fixes the issue, but I'm not able to get it to work with `1.8.0` at all.

--- a/docs/faq.md
+++ b/docs/faq.md
@@ -545,15 +545,14 @@ Sometimes when upgrading your gradle build plugin versions, you may experience e
 This commonly appears in multi module gradle projects. A solution to this problem is to define all of your plugins in the base project and apply them selectively in your subprojects as needed. This should help alleviate the problem of the buildscript classpath using older versions of a library.
 
 `build.gradle` (root)
-```
+```groovy
 plugins {
   id 'com.google.cloud.tools.jib' version 'x.y.z' apply false
-  ...
 }
 ```
 
 `build.gradle` (sub-project)
-```
+```groovy
 plugins {
   id 'com.google.cloud.tools.jib'
 }

--- a/docs/faq.md
+++ b/docs/faq.md
@@ -540,7 +540,7 @@ See more at [Using Google Container Registry (GCR) with Minikube](https://ryanes
 
 ### I am seeing `Method Not Found` or `Class Not Found` errors when building.
 
-Sometimes when upgrading your gradle build plugin versions, you may experience errors due to mismatching versions of dependencies pulled in. For example: [issues/2183](https://github.com/GoogleContainerTools/jib/issues/2183). This can be due to the buildscript classpath loading behavior described [on gradle forums](https://discuss.gradle.org/t/version-is-root-build-gradle-buildscript-is-overriding-subproject-buildscript-dependency-versions/20746/3). 
+Sometimes when upgrading your gradle build plugin versions, you may experience errors due to mismatching versions of dependencies pulled in (for example: [issues/2183](https://github.com/GoogleContainerTools/jib/issues/2183)). This can be due to the buildscript classpath loading behavior described [on gradle forums](https://discuss.gradle.org/t/version-is-root-build-gradle-buildscript-is-overriding-subproject-buildscript-dependency-versions/20746/3). 
 
 This commonly appears in multi module gradle projects. A solution to this problem is to define all your plugins in the base project and apply them selectively in your subprojects as needed. This should help alleviate the problem of the buildscript classpath using older versions of a library.
 

--- a/docs/faq.md
+++ b/docs/faq.md
@@ -547,7 +547,7 @@ This commonly appears in multi module gradle projects. A solution to this proble
 `build.gradle` (root)
 ```
 plugins {
-  id `com.google.cloud.tools.jib` version `x.y.z` apply false
+  id 'com.google.cloud.tools.jib' version 'x.y.z' apply false
   ...
 }
 ```
@@ -555,7 +555,7 @@ plugins {
 `build.gradle` (sub-project)
 ```
 plugins {
-  id `com.google.cloud.tools.jib`
+  id 'com.google.cloud.tools.jib'
 }
 ```
 

--- a/docs/faq.md
+++ b/docs/faq.md
@@ -37,7 +37,7 @@ If a question you have is not answered below, please [submit an issue](/../../is
 [How can I examine network traffic?](#how-can-i-examine-network-traffic)\
 [How do I view debug logs for Jib?](#how-do-i-view-debug-logs-for-jib)\
 [I am seeing `ImagePullBackoff` on my pods.](#i-am-seeing-imagepullbackoff-on-my-pods-in-minikube)\
-[I am seeing `Method Not Found` or `Class Not Found` when building](#i-am-seeing-method-not-found-or-class-not-found-when-building)
+[I am seeing `Method Not Found` or `Class Not Found` errors when building.](#i-am-seeing-method-not-found-or-class-not-found-errors-when-building)
 
 ---
 
@@ -538,7 +538,7 @@ kubectl patch serviceaccount default \
 
 See more at [Using Google Container Registry (GCR) with Minikube](https://ryaneschinger.com/blog/using-google-container-registry-gcr-with-minikube/).
 
-### I am seeing `Method Not Found` or `Class Not Found` when building
+### I am seeing `Method Not Found` or `Class Not Found` errors when building.
 
 Sometimes when upgrading your gradle build plugin versions, you may experience errors due to mismatching versions of dependencies pulled in. For example: [issues/2183](https://github.com/GoogleContainerTools/jib/issues/2183). This can be due to the buildscript classpath loading behavior described [on gradle forums](https://discuss.gradle.org/t/version-is-root-build-gradle-buildscript-is-overriding-subproject-buildscript-dependency-versions/20746/3). 
 

--- a/docs/faq.md
+++ b/docs/faq.md
@@ -36,7 +36,8 @@ If a question you have is not answered below, please [submit an issue](/../../is
 [How do I configure a proxy?](#how-do-i-configure-a-proxy)\
 [How can I examine network traffic?](#how-can-i-examine-network-traffic)\
 [How do I view debug logs for Jib?](#how-do-i-view-debug-logs-for-jib)\
-[I am seeing `ImagePullBackoff` on my pods.](#i-am-seeing-imagepullbackoff-on-my-pods-in-minikube)
+[I am seeing `ImagePullBackoff` on my pods.](#i-am-seeing-imagepullbackoff-on-my-pods-in-minikube)\
+[I am seeing `Method Not Found` or `Class Not Found` when building](#i-am-seeing-method-not-found-or-class-not-found-when-building)
 
 ---
 
@@ -537,6 +538,26 @@ kubectl patch serviceaccount default \
 
 See more at [Using Google Container Registry (GCR) with Minikube](https://ryaneschinger.com/blog/using-google-container-registry-gcr-with-minikube/).
 
+### I am seeing `Method Not Found` or `Class Not Found` when building
+
+Sometimes when upgrading your gradle build plugin versions, you may experience errors due to mismatching versions of dependencies pulled in. For example: [issues/2183](https://github.com/GoogleContainerTools/jib/issues/2183). This can be due to the buildscript classpath loading behavior described [on gradle forums](https://discuss.gradle.org/t/version-is-root-build-gradle-buildscript-is-overriding-subproject-buildscript-dependency-versions/20746/3). 
+
+This commonly appears in multi module gradle projects. A solution to this problem is to define all your plugins in the base project and apply them selectively in your subprojects as needed. This should help alleviate the problem of the buildscript classpath using older versions of a library.
+
+`build.gradle` (root)
+```
+plugins {
+  id `com.google.cloud.tools.jib` version `x.y.z` apply false
+  ...
+}
+```
+
+`build.gradle` (sub-project)
+```
+plugins {
+  id `com.google.cloud.tools.jib`
+}
+```
 
 ### How can I examine network traffic?
 

--- a/docs/faq.md
+++ b/docs/faq.md
@@ -542,7 +542,7 @@ See more at [Using Google Container Registry (GCR) with Minikube](https://ryanes
 
 Sometimes when upgrading your gradle build plugin versions, you may experience errors due to mismatching versions of dependencies pulled in (for example: [issues/2183](https://github.com/GoogleContainerTools/jib/issues/2183)). This can be due to the buildscript classpath loading behavior described [on gradle forums](https://discuss.gradle.org/t/version-is-root-build-gradle-buildscript-is-overriding-subproject-buildscript-dependency-versions/20746/3). 
 
-This commonly appears in multi module gradle projects. A solution to this problem is to define all your plugins in the base project and apply them selectively in your subprojects as needed. This should help alleviate the problem of the buildscript classpath using older versions of a library.
+This commonly appears in multi module gradle projects. A solution to this problem is to define all of your plugins in the base project and apply them selectively in your subprojects as needed. This should help alleviate the problem of the buildscript classpath using older versions of a library.
 
 `build.gradle` (root)
 ```

